Jenkinsfile

```diff
@@ -97,8 +97,8 @@ def functionalDynamicParallelSteps(image){
         def currentCiGroup = "ciGroup${i}";
         def currentStep = i;
         ciGroupsMap["${currentCiGroup}"] = {
-            sh "rm -rf ${env.WORKSPACE}_${currentCiGroup}"
-            sh "mkdir ${env.WORKSPACE}_${currentCiGroup}"
+            sh "rm -rf ${env.WORKSPACE}/.optimize/${currentCiGroup}"
+            sh "mkdir ${env.WORKSPACE}/.optimize/${currentCiGroup}"
```

Comment:
You might want to use -p. Otherwise it could fail if running for the first time, when .optimize directory does not exist

Reply:
mkdir -p

Reply:
-p option has been added to ensure directory path.

Test job has been run: https://jenkins.bfs.sichend.people.aws.dev/blue/organizations/jenkins/Kibana/detail/bfs7.7.1_test/18/pipeline/

Reply:
rm -rf /var/lib/jenkins/workspace/Kibana_bfs7.7.1_test_ciGroup9

This is not right in the test run.

Reply:
Hi, @sichend. I sent the wrong link. Here is the correct link: https://jenkins.bfs.sichend.people.aws.dev/blue/organizations/jenkins/Kibana/detail/bfs7.7.1_test/19/pipeline/60/